src/pages/listpage/CardListPage.scss

```
@@ -0,0 +1,26 @@
+@import 'assets/styles/common.scss';
+@import 'assets/styles/fonts.scss';
+@import 'assets/styles/palette.scss';
+
+.cardlistlayer {
+  padding: 50px 0;
+  @include Flex(column, center, center, '');
+  .cardlistbox {
+    width: 1200px;
+    @include Flex(column, center, center, '');
+    p {
+      margin-bottom: 16px;
+      @include font-style(24, bold);
+      line-height: 36px;
+      font-weight: 700;
+    }
+    .datecardlist {
+      margin-bottom: 40px;
+      @include Flex(row, center, center, 20px);
+    }
+    .hotcardlist {
+      @include Flex(row, center, center, 20px);
+      margin-bottom: 50px;
+    }
```

Flex 믹스인이 공통적이기 때문에 클래스명을 두개 주는건 어떨까요? 공통적인 card-list클래스와 date-card, hot-card를 각각 주는거죵 두 부분이 공통적인 부분이 많다 생각돼서 공통 클래스명 주는게 좋을거같네용~!
```suggestion
.card-list {
  @include Flex(row, center, center, 20px);
  
  &.date-card {
    margin-bottom: 40px;
  }
  &.hot-card {
    margin-bottom: 50px;
  }
}
```